component/constraint/CableConstraint.h

```diff
@@ -148,6 +148,7 @@ class CableConstraint : public CableModel<DataTypes>
     using CableModel<DataTypes>::d_maxDispVariation ;
     using CableModel<DataTypes>::d_force ;
     using CableModel<DataTypes>::d_displacement ;
+//    using CableModel<DataTypes>::m_componentstate ;
```

Comment:
Here the using does not compile, and I don't understand why...  
Note that it works in CableModel

Here's the error:
`erreur : ‘sofa::core::objectmodel::ComponentState sofa::core::objectmodel::BaseObject::m_componentstate’ is protected`